tests/helpers/linux/test_tc.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) ByteDance, Inc. and its affiliates.
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import os
+from random import choice
+from string import ascii_letters
+import unittest
+
+from drgn.helpers.linux.fs import path_lookup
+from drgn.helpers.linux.net import get_net_ns_by_inode, netdev_get_by_name
+from drgn.helpers.linux.tc import qdisc_lookup
+from tests.helpers.linux import LinuxHelperTestCase
+
+try:
+    from pr2modules.netlink.exceptions import NetlinkError
+    from pyroute2 import NetNS
+
+    pyroute2 = True
```

Comment:
Can we call this `have_pyroute2` to avoid ambiguity with the module name itself?